integration/tests/user_scenario/basic_workflow/negative/neg_specify_deleted_forge_module.rb

```diff
@@ -0,0 +1,45 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
```

Comment:
Minor quibble: no methods from the "master_manipulator" library are actually use in this test. Doesn't affect anything.